clang/include/clang/Basic/OffloadArch.h

```diff
@@ -102,13 +102,14 @@ enum class OffloadArch {
   Generic, // A processor model named 'generic' if the target backend defines a
            // public one.
   // Intel CPUs
-  GRANITERAPIDS,
+  GRANITERAPIDS_CPU,
```

Comment:
Is this the format we will follow going forward? Processor name + "_" + CPU/GPU? I am ok with it.
Adding @bader for comment.

Thanks